### PR TITLE
Added new text explaining mAst and new mLum check (MaxCLL/MaxFall)

### PR DESCRIPTION
**Pull request by digitaltvguy**
I hope I got this right.  It's a start.

### CODE REVIEW

File: index.html
Comment (ProgramMax) [inline index.html:4097]: "the mastering display's" with an apostrophe, right?

File: index.html
Comment (ProgramMax) [inline index.html:4099]: "frame's average" should also have an apostrophe.

File: index.html
Comment (ProgramMax) [inline index.html:4106]: mLUm

File: index.html
Comment (ProgramMax) [inline index.html:4130]: mLUm

File: index.html
Comment (ProgramMax) [inline index.html:4133]: This isn't critical, but this line should be indented by 2 spaces.

File: index.html
Comment (ProgramMax) [inline index.html:4135]: mLUm

File: index.html
Comment (ProgramMax) [inline index.html:4150]: mLUm

File: index.html
Comment (ProgramMax) [inline index.html:4158]: 250 cd/m^2 should have the stored value of 2500000, right?

**Review by ProgramMax — CHANGES_REQUESTED** [review]
Thank you for doing this.

I've been sick recently and you beat me to it. :)

The commentary you added is very valuable. Thank you for that, too. Glad you're able to share your wisdom.

There are a few things which should be changed before this lands. But they're small, simple things.

File: index.html
Comment (chrisn) [reply index.html:3859]: If Mastering Display Color Volume is defined in the SMPTE spec, does it need quotes around it?

File: index.html
Comment (svgeesus) [inline index.html:4086]: I had [suggested]()  Maximum and Frame Average light levels as the section title, which I think is clearer. "mastering metadata" is vague and applies equally to mASt.

Comment (digitaltvguy) [reply index.html:4086]: How about "Metadata for Maximum Single-Pixel  and Frame-Average Luminance Levels". Since this is a still graphic, there is only a single frame.  Maybe the description can identify this

File: index.html
Comment (digitaltvguy) [reply index.html]: done

File: index.html
Comment (digitaltvguy) [reply index.html]: Done

File: index.html
Comment (digitaltvguy) [reply index.html]: Done

File: index.html
Comment (digitaltvguy) [reply index.html]: Done

File: index.html
Comment (digitaltvguy) [reply index.html:4154]: Done

File: index.html
Comment (digitaltvguy) [reply index.html]: Done

File: index.html
Comment (digitaltvguy) [reply index.html]: Done

File: index.html
Comment (digitaltvguy) [reply index.html:4179]: Done

File: index.html
Comment (digitaltvguy) [reply index.html]: Done

File: index.html
Comment (ProgramMax) [reply index.html]: Since APNG is now supported, it is no longer a single frame.

Comment (digitaltvguy) [reply index.html]: I've added a description with an exception for multiple frames (image sequence).

**Review by svgeesus — APPROVED** [review]
I think this looks pretty good now.
We might want to tighten the integration with APNG a little, to improve testability, but that can be a separate issue.

**Comment by svgeesus** [comment]
@ProgramMax please re-review, it looks like your requested changes were made?

File: index.html
Comment (ProgramMax) [reply index.html:4182]: I'm still seeing the stored value of 250.
I think we want the actual value of 250 cd/m^2 and the stored value of 2500000.

**Review by ProgramMax — APPROVED** [review]
Thank you for updating it.
There are a few more changes which will need to be made. But I'm approving the pull request because those changes can be done in a follow-up. I don't want to hold up your work.

**Comment by svgeesus** [comment]
Merged, [clean-up PR here](https://github.com/w3c/PNG-spec/pull/258)

**Pull request merged**
